CalibCalorimetry/HcalPlugins/python/Hcal_Conditions_forGlobalTag_cff.py

```diff
@@ -114,7 +114,10 @@
             intlumiOffset = cms.double(150),
             depVsTemp = cms.double(0.0631),
             intlumiToNeutrons = cms.double(3.67e8),
-            depVsNeutrons = cms.vdouble(5.69e-11,7.90e-11),
+            # slopes taken from https://twiki.cern.ch/twiki/pub/CMSPublic/HcalDPGResultsCMSDPS2017042/dark_current_vs_lumi.png
+            # crosstalk divided out
+            # heUpgrade intlumiToNeutrons factor (below) used to convert from /fb-1 to /neutrons 
+            depVsNeutrons = cms.vdouble(5.543e-10,8.012e-10),
```

Comment:
This is a factor 10 larger than the previous values: please confirm that it is correct
(How can I get these numbers from what in https://twiki.cern.ch/twiki/pub/CMSPublic/HcalDPGResultsCMSDPS2017042/dark_current_vs_lumi.png)

Reply:
Indeed, the prediction has increased by a factor of 10. (This necessitated the re-evaluation of PF thresholds. The previous prediction came from a much older measurement.)

You can get the numbers from the plot by taking the slopes, multiplying by `(1-lambda)` (`lambda` is given as `crosstalk` in the `SiPMCharacteristics` section further down in this file; 0.17 or 0.196 for the different SiPM types), and dividing by the HE dose value `intlumiToNeutrons = cms.double(2.92e7)` (which is in fb-1/neutrons). This is indicated in the comment.